content/zh-cn/docs/concepts/configuration/configmap.md

```diff
@@ -48,15 +48,18 @@ debug the exact same code locally if needed.
 这让你可以获取在云中运行的容器镜像，并且如果有需要的话，在本地调试完全相同的代码。
 
 <!--
+{{< note >}}
 A ConfigMap is not designed to hold large chunks of data. The data stored in a
 ConfigMap cannot exceed 1 MiB. If you need to store settings that are
 larger than this limit, you may want to consider mounting a volume or use a
 separate database or file service.
+{{< /note >}}
 -->
+{{< note >}}
```

Comment:
move this line before line 50, and remove line 51 and 56, please.

Reply:
Done